recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "openmm" %}
 {% set version = "7.4.0" %}
-{% set build = 4 %}
+{% set build = 5 %}
```

Comment:
This should be increased to `6`, since the previous PR got built and uploaded to anaconda.org.